charts/incubator/misskey/questions.yaml

```diff
@@ -0,0 +1,161 @@
+# Include{groups}
+portals:
+  open:
+# Include{portalLink}
+questions:
+# Include{global}
+# Include{controller}
+# Include{controllerDeployment}
+# Include{replicas}
+# Include{replica1}
+# Include{strategy}
+# Include{recreate}
+# Include{controllerExpert}
+# Include{controllerExpertExtraArgs}
+  - variable: misskey
+    group: "Container Configuration"
+    label: "Misskey Configuration"
+      schema:
+        additional_attrs: true
+        type: dict
+        attrs:
+          - variable: url
+            label: "Final Accessible URL"
+            description: "Final accessible URL seen by a user. ONCE YOU HAVE STARTED THE INSTANCE, DO NOT CHANGE THE URL SETTINGS AFTER THAT!"
+            schema:
+              type: string
+              required: true
+          - variable: id
+            label: "ID Generation Method"
+            description: "Select the ID generation method. DO NOT CHANGE AFTER INSTANCE IS STARTED!"
+            schema:
+              type: string
+              enum:
+                - value: "aid"
+                  description: "Short, Millisecond accuracy"
+                - value: "meid"
+                  description: "Similar to ObjectID, Millisecond accuracy"
+                - value: "ulid"
+                  description: "Millisecond accuracy"
+                - value: "objectid"
+                  description: "This is left for backward compatibility"
+              default: "aid"
+              required: true
+# Include{containerConfig}
+# Include{serviceRoot}
+        - variable: main
+          label: "Main Service"
+          description: "The Primary service on which the healthcheck runs, often the webUI"
+          schema:
+            additional_attrs: true
+            type: dict
+            attrs:
+# Include{serviceSelectorLoadBalancer}
+# Include{serviceSelectorExtras}
+                    - variable: main
+                      label: "Main Service Port Configuration"
+                      schema:
+                        additional_attrs: true
+                        type: dict
+                        attrs:
+                          - variable: port
+                            label: "Port"
+                            description: "This port exposes the container port on the service"
+                            schema:
+                              type: int
+                              default: 3003
+                              required: true
+# Include{advancedPortHTTP}
+                                - variable: targetPort
+                                  label: "Target Port"
+                                  description: "The internal(!) port on the container the Application runs on"
+                                  schema:
+                                    type: int
+                                    default: 3003
```

Comment:
Don't change the target port... leave it as `3000` as its the port used internally.